### ISSUE TITLE
Code Code Code

### ISSUE THREAD

**Issue by PeterWron** [HumanAIGC/OutfitAnyone#39]

**Comment by East91221** [comment]
![blaz](https://github.com/HumanAIGC/OutfitAnyone/assets/156676585/c6b0a171-e83a-4f1d-9293-58c3ba5c2efe)
![User](https://github.com/HumanAIGC/OutfitAnyone/assets/156676585/05801a87-c410-4ca0-92f6-ddbe44430cbf)
![lower](https://github.com/HumanAIGC/OutfitAnyone/assets/156676585/6e6a3dcf-553b-4090-8a97-dd35d6f7c05e)
![image (15)](https://github.com/HumanAIGC/OutfitAnyone/assets/156676585/93bc69a1-3dc4-4b61-a8cd-a8ad95462c49)
![image (19)](https://github.com/HumanAIGC/OutfitAnyone/assets/156676585/0d4d5fb6-a871-4c70-ae5f-3620de3d2412)
![Result1](https://github.com/HumanAIGC/OutfitAnyone/assets/156676585/4518f54b-8a44-4883-b211-8108921f2a42)
![Result2](https://github.com/HumanAIGC/OutfitAnyone/assets/156676585/274bb15e-3b27-442f-b87c-d94869d10294)

**Comment by East91221** [comment]
Are you mad? Kidding? They never release code. They already mentioned as it's purely for research purpose. Even though, They release, It doesn't work as you needed. They are most insecure people. They afraid as other may develop the same.

You can try their previous repos by studying how they developed outfit anyone so far. They used the zero shot try on along with post hoc refine...etc. The work consist of a couple of different stable diffusions combination. They modified the existing models available. They did NOT developed from scratch. Follow their previous repos. You get how to do. It's simple.

I tried and I got the results. I followed the paper and tried multiple repos. Remember, The work is NOT developed by training on dataset. I attach a sample here. have a look. You would become grand father by the time you see the code working.

**Comment by East91221** [comment]
try yourself or else keep looking at default models using clothing images. Read carefully. They already mentioned that it's for research. They never release code. 

If they release, many AI developers like me would replicate in few days though code doesn't work. They stick to their policies and principles. You become grand father till you see the code working.

For now, Just Enjoy. It's NOT waste of Github resource. 

Try to bring or pull the best by using their previous repos like ClothtoTex and other similar repos with combine diffusion models..

Do you need the code to see for free???? What you do? You make it available for commercial? What's next? Making Money? That's what they restrict. It's purely for research and they are DONE. They will move on to other kind of development. That's ALL

**Comment by East91221** [comment]
@East91221 Thankyou for still NOT releasing as others make money which is against to your principles.

**Comment by East91221** [comment]
@HumanAIGC Thankyou for still NOT releasing as others make money which is against to your principles.

**Comment by yimingz1218** [comment]
> try yourself or else keep looking at default models using clothing images. Read carefully. They already mentioned that it's for research. They never release code.
> 
> If they release, many AI developers like me would replicate in few days though code doesn't work. They stick to their policies and principles. You become grand father till you see the code working.
> 
> For now, Just Enjoy. It's NOT waste of Github resource.
> 
> Try to bring or pull the best by using their previous repos like ClothtoTex and other similar repos with combine diffusion models..
> 
> Do you need the code to see for free???? What you do? You make it available for commercial? What's next? Making Money? That's what they restrict. It's purely for research and they are DONE. They will move on to other kind of development. That's ALL

It's very simple. If they do not release the model, other researchers would not know how GOOD the model is and what artifects it has. Then it is very difficult to carry on the research in the area of VITON since the seemingly SOTA model only releases a few cherry-picked results. Of course, as a project that doesn't even have an arxiv paper, researchers can just ignore it and never compare with it in their papers. Then what is the contribution of this paper for the research community? Nothing. Just a few cherry-picked results which caught the attention of publics in a few days. 

It is everybody's freedom not to release their code. And you cannot blame someone if they don't release their code. But if all researchers tend not to opensource their work, then there will be no progress in technology. See what those amazing opensourced works from 2012 to now? That's why AI is growing so wildly.

**Comment by minssjeon** [comment]
Hello. I'm a company that does fashion consulting and related business in Korea. I would like to use your program, and I would appreciate it if you could let me know what procedures I should go through. I'm writing like this because I couldn't find the e-mail information. Please reply to my post and I will contact you right away.

**Comment by Vvikram129** [comment]
> > try yourself or else keep looking at default models using clothing images. Read carefully. They already mentioned that it's for research. They never release code.
> > If they release, many AI developers like me would replicate in few days though code doesn't work. They stick to their policies and principles. You become grand father till you see the code working.
> > For now, Just Enjoy. It's NOT waste of Github resource.
> > Try to bring or pull the best by using their previous repos like ClothtoTex and other similar repos with combine diffusion models..
> > Do you need the code to see for free???? What you do? You make it available for commercial? What's next? Making Money? That's what they restrict. It's purely for research and they are DONE. They will move on to other kind of development. That's ALL
> 
> It's very simple. If they do not release the model, other researchers would not know how GOOD the model is and what artifects it has. Then it is very difficult to carry on the research in the area of VITON since the seemingly SOTA model only releases a few cherry-picked results. Of course, as a project that doesn't even have an arxiv paper, researchers can just ignore it and never compare with it in their papers. Then what is the contribution of this paper for the research community? Nothing. Just a few cherry-picked results which caught the attention of publics in a few days.
> 
> It is everybody's freedom not to release their code. And you cannot blame someone if they don't release their code. But if all researchers tend not to opensource their work, then there will be no progress in technology. See those amazing opensourced works from 2012 to now? That's why AI is growing so wildly.

Hi. I agree with the unknown person post. He is right. They do not release.

**Comment by ShawnFumo** [comment]
> Are you mad? Kidding? They never release code.

Not true. They didn't for a few there, but for instance this newer project Cloth2Tex both has an arXiv paper and some code: https://tomguluson92.github.io/projects/cloth2tex/
https://github.com/HumanAIGC/Cloth2Tex
